SU-SWS/stanford_news
# D8CORE-2110: Removed news views paragraph type.

# READY FOR REVIEW

# Summary
- Removes the news views paragraphs

# Needed By (Date)
- asap

# Urgency
- top

# Steps to Test

1. Review code to ensure no more news views paragraph config is in the module
(optional as stack builds are forthcoming)
2. Do a fresh build of cardinalsites with the 8.1.8 release of stanford_profile
3. Enable this module
4. Ensure no paragraph type is available
